compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -5078,3 +5079,76 @@ declare_lint! {
     };
     crate_level_only
 }
+
+declare_lint! {
+    /// The `abi_unsupported_vector_types` lint detects function definitions and calls
+    /// whose ABI depends on enabling certain target features that do not enable those features.
```

Comment:
```suggestion
    /// whose ABI depends on enabling certain target features, but those features are not enabled.
```
I found the sentence very hard to read otherwise